app/federation/server/_client/callbacks/afterCreateRoom.js

```diff
@@ -0,0 +1,82 @@
+import { callbacks } from '../../../../callbacks';
+import { logger } from '../../logger';
+import { FederationRoomEvents, Subscriptions, Users } from '../../../../models/server';
+import { Federation } from '../../federation';
+import { normalizers } from '../../normalizers';
+import { deleteRoom } from '../../../../lib/server/functions';
+
+async function run(room, users, subscriptions) {
```

Comment:
this function can be renamed to `doAfterCreateRoom` and exported directly from here:
```suggestion
export async function doAfterCreateRoom(room, users, subscriptions) {
```